docs/docs/features/hardware-transcoding.md

```diff
@@ -42,13 +42,24 @@ As this is a new feature, it is still experimental and may not work on all syste
   - Additionally, if the server specifically has an 11th gen CPU and is running kernel 5.15 (shipped with Ubuntu 22.04 LTS), then you will need to upgrade this kernel (from [Jellyfin docs][jellyfin-kernel-bug])
 
 ## Setup
-
+#### Initial Setup
 1. If you do not already have it, download the latest [`hwaccel.yml`][hw-file] file and ensure it's in the same folder as the `docker-compose.yml`.
 2. Uncomment the lines that apply to your system and desired usage.
 3. In the `docker-compose.yml` under `immich-microservices`, uncomment the lines relating to the `hwaccel.yml` file.
 4. Redeploy the `immich-microservices` container with these updated settings.
 5. In the Admin page under `FFmpeg settings`, change the hardware acceleration setting to the appropriate option and save.
 
+#### All-In-One - Unraid Setup
+##### NVENC - NVIDIA GPUs
+- If you are using other backends. You will still need to implement [`hwaccel.yml`][hw-file] file into the `immich-microservices` service directly, please see the "Initial Setup" section above on how to do that.
+- Steps 1 and 2 have been now implemented in the .env or environment variables of your docker-compose. If you are missing them from your config, please add them. Otherwise, skip to step 3.    
```

Comment:
The envs are actually in the docker image itself now, not in the .env, meaning they'll be present for everyone. But since this change will only come in the next release, maybe phrase it as "As of v1.92.0, steps 1 and 2 are no longer necessary. If your version of Immich is below that, please follow these steps. Otherwise, skip to step 3."